src/rules/selector-attribute-quotes/index.js

```diff
@@ -0,0 +1,70 @@
+import {
+  isStandardSyntaxRule,
+  isStandardSyntaxSelector,
+  parseSelector,
+  report,
+  ruleMessages,
+  validateOptions,
+} from "../../utils"
+
+export const ruleName = "selector-attribute-quotes"
+
+export const messages = ruleMessages(ruleName, {
+  expectedQuotes: "Expected quotes",
+  rejectedQuotes: "Unexpected quotes",
+})
+
+function isQuotesExist(str) {
```

Comment:
I think this function is unnecessary. We can simply use `postcss-selector-parser`'s [`attribute.quoted`](https://github.com/postcss/postcss-selector-parser/blob/master/API.md#attributequoted) in the conditionals instead.